_includes/pbc/all/install-features/202410.0/install-the-product-feature.md

```diff
@@ -353,6 +353,56 @@ Make sure that you can manage "multiselect" product attributes at `https://zed.d
 
 {% endinfo_block %}
 
+### 5) Set up the product concrete table filter
```

Comment:
If the doc does not exist in the 202505.0 folder yet, he 202410.0 version must be copied to 202505.0 and the 202505.0 version to be extended by your changes